pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
https://github.com/opencivicdata/scrapers-ca/ uses pupa outside a Django project and has no other settings except for `pupa_settings.py`.